autogluon/task/tabular_prediction/predictor.py

```diff
@@ -386,6 +388,43 @@ def feature_importance(self, model=None, dataset=None, features=None, raw=True,
 
         return self._learner.get_feature_importance(model=model, X=dataset, features=features, raw=raw, subsample_size=subsample_size, silent=silent)
 
+    def refit_full(self, model=None):
+        """
+        Optimizes a model's inference time by collapsing bagged ensembles into a single model fit on all of the training data.
+        For bagged models, this process will typically result in a slight accuracy reduction and a large inference speedup.
+            The inference speedup will generally be between 10-200x faster than the original bagged ensemble model.
```

Comment:
Why not define the speedup mathematically, is the approximation too crude?
E.g. I'd imagine with k-fold bagging, the full model will be ~k times faster than the bagged model (also for repeated bagging, etc.).

Reply:
Originally I thought this would be too technical, but I think its manageable, added.